qfieldsync/gui/checker_feedback_table.py

```diff
@@ -54,7 +45,16 @@ def __init__(self, checker_feedback: ProjectCheckerFeedback, *args, **kwargs):
                 self.insertRow(row)
 
                 # first column
-                item = QTableWidgetItem(str(feedback.level.name))
+                if feedback.level == Feedback.Level.WARNING:
+                    level_icon = "/mIconWarning.svg"
+                    level_text = self.tr("Warning")
+                else:
+                    level_icon = "/mIconCritical.svg"
+                    level_text = self.tr("Error")
+
+                item = QTableWidgetItem(
+                    QgsApplication.getThemeIcon(level_icon), level_text
```

Comment:
I wonder if we want to make the text just a tooltip, I don't see us having any benefit of this in written form here

Reply:
One benefit is that it ties in with the explainer above. I'd be keen to see it stay.